src/ZendeskApi.Client/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -1,5 +1,8 @@
+using System;
+using System.Net.Http;
 using Microsoft.Extensions.DependencyInjection;
 using ZendeskApi.Client.Options;
+#pragma warning disable 618
```

Comment:
Why's this added?

Reply:
Because the old ZendeskApiClient is marked as Obsolete and produces a warning which we don't want as we'll remove it eventually.